L1Trigger/TrackTrigger/src/L1TrackQuality.cc

```diff
@@ -90,7 +90,7 @@ std::vector<float> L1TrackQuality::featureTransform(TTTrack<Ref_Phase2TrackerDig
   feature_map["bendchi2_bin"] = tmp_trk_bendchi2_bin;
   feature_map["chi2rphi_bin"] = tmp_trk_chi2rphi_bin;
   feature_map["chi2rz_bin"] = tmp_trk_chi2rz_bin;
-  feature_map["tanl"] = tmp_trk_tanl;
```

Comment:
Why is this being removed? Doesn't the prompt classifier need this?

Reply:
Just added this back in commit [dbf8525](https://github.com/cms-l1t-offline/cmssw/pull/1257/commits/dbf852576508f471657cead5177e4066460d8381)